pkg/cmd/env.go

```diff
@@ -166,6 +170,10 @@ func CreateOrUpdateEnv(ctx context.Context, envArgs *types.EnvMeta, args []strin
 	if err != nil {
 		return err
 	}
+	if err := c.Create(ctx, &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: envArgs.Namespace}}); err != nil && !apierrors.IsAlreadyExists(err) {
```

Comment:
Checking whether the namespace exists before creating is the other option, which one is better?

Reply:
Haha, I think one rpc request win